src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -21,54 +23,97 @@ type TableRowData = RowData & {
 
 // Columns
 
+function cellDefaultTooltip(displayValue: string) {
+  return <Tooltip title={displayValue}>{displayValue}</Tooltip>;
+}
+
+function cellNullTooltip() {
+  return (
+    <Tooltip title="NULL">
+      <i>NULL</i>
+    </Tooltip>
+  );
+}
+
+function cellNumericGeneralizedTooltip(binSize: number, v: number) {
+  const displayValue = v.toString();
```

Comment:
Not required to be converted to string. `<Tooltip title={tooltip}>{v}</Tooltip>` should work?